core/jobs_test.py

```diff
@@ -92,12 +129,30 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Implements the map function. Must be declared @staticmethod.
+
+        Args:
+            item: *. A single element of type given by entity_class().
+
+        Yields:
+            tuple(str,int). One might yield (exploration.id, 1).
```

Comment:
Here and elsewhere, please change to:
```suggestion
            tuple(str,int). Yields (exploration.id, 1).
```